src/schema/v2/me/sendIdentityVerficationEmailMutation.ts

```diff
@@ -88,14 +93,14 @@ export const sendIdentityVerificationEmailMutation = mutationWithClientMutationI
     },
   },
   mutateAndGetPayload: (
-    { userID },
+    { userID, email },
     { sendIdentityVerificationEmailLoader }
   ) => {
     if (!sendIdentityVerificationEmailLoader) {
       throw new Error("You need to be signed in to perform this action")
     }
 
-    return sendIdentityVerificationEmailLoader(userID)
+    return sendIdentityVerificationEmailLoader({ user_id: userID, email })
```

Comment:
@SamRozen noting this change means we will no longer be relying on this block to [execute](https://github.com/artsy/gravity/blob/cdb23413136292713f487c03f21a3ad821d5b6fd/app/api/v1/identity_verifications_endpoint.rb#L62-L66), but instead the first block with the `user_id` [here](https://github.com/artsy/gravity/blob/cdb23413136292713f487c03f21a3ad821d5b6fd/app/api/v1/identity_verifications_endpoint.rb#L51:L56). Was this the original intention?

Reply:
I'm not sure this will work as currently implemented. The API endpoint [only authorizes](https://github.com/artsy/gravity/blob/0b6ab6fc4de189d1d693940a1029649e0d99256a/app/models/domain/ability.rb#L200-L202) certain administrators to create verifications for arbitrary user IDs, but [that fall-through logic](https://github.com/artsy/gravity/blob/cdb23413136292713f487c03f21a3ad821d5b6fd/app/api/v1/identity_verifications_endpoint.rb#L62-L66) also allows users to initiate verifications _for themselves only_. For this to work, we could modify Gravity's authorization logic to also permit the case when `params['user_id'] == current_user.id`.

Reply:
@SamRozen - Joey just pointed out I tagged you instead of @rajsam003. Sorry for the confusion! 

@rajsam003 - let me know if you see any issue with the above. Will also have a `gravity` change.

Reply:
gravity PR: https://github.com/artsy/gravity/pull/15430